From 89c8b675ffbfbbade7900d33ab2e308b25d316ed Mon Sep 17 00:00:00 2001
From: Danny Lin <danny@kdrag0n.dev>
Date: Fri, 25 Sep 2020 12:08:23 -0700
Subject: [PATCH 2/15] SystemUI: Fix uneven volume icon padding in status bar

The current silent and vibrate volume icons in the status bar have
slightly too much padding compared to all of the other status bar icons,
which makes it look out-of-place. Remove the excess insets to make it
more even.

Note that the silent icon still has 0.5 dp of padding on its left and
right sides to make its total size match that of the vibrate icon, which
is 19 dp, while the silent icon is 18 dp.

Change-Id: I7fb9a6e0113f6103eed308ac2d25c392fe8abb73
---
 packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml  | 4 ++--
 packages/SystemUI/res/drawable/stat_sys_ringer_vibrate.xml | 6 +++---
 2 files changed, 5 insertions(+), 5 deletions(-)

diff --git a/packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml b/packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml
index a8f0cc3a1d92..1a0599107f69 100644
--- a/packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_ringer_silent.xml
@@ -14,8 +14,8 @@ Copyright (C) 2015 The Android Open Source Project
     limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-        android:insetLeft="3dp"
-        android:insetRight="3dp">
+        android:insetLeft="0.5dp"
+        android:insetRight="0.5dp">
     <vector android:width="18dp"
             android:height="18dp"
             android:viewportWidth="24.0"
diff --git a/packages/SystemUI/res/drawable/stat_sys_ringer_vibrate.xml b/packages/SystemUI/res/drawable/stat_sys_ringer_vibrate.xml
index 21a4c1703d31..5d196846822a 100644
--- a/packages/SystemUI/res/drawable/stat_sys_ringer_vibrate.xml
+++ b/packages/SystemUI/res/drawable/stat_sys_ringer_vibrate.xml
@@ -14,6 +14,6 @@
      limitations under the License.
 -->
 <inset xmlns:android="http://schemas.android.com/apk/res/android"
-    android:insetLeft="2.5dp"
-    android:insetRight="2.5dp"
-    android:drawable="@drawable/ic_volume_ringer_vibrate" />
\ No newline at end of file
+    android:insetLeft="0dp"
+    android:insetRight="0dp"
+    android:drawable="@drawable/ic_volume_ringer_vibrate" />
-- 
2.25.1

